stix_shifter_modules/azure_log_analytics/configuration/lang_en.json

```diff
@@ -0,0 +1,44 @@
+{
+  "connection": {
+    "host": {
+      "label": "Management IP address or Hostname",
+      "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
```

Comment:
Just leave as `Specify the IP address or hostname of the data source` to keep it generic.

Reply:
we have to do clean up work on all connectors to remove the IBM platform name. will remove this one